release_draft.md

```diff
@@ -0,0 +1,299 @@
+ign or gz, ign/gz is used for this propose) `foo` with major version `X`:
+
+ 1. [`release.py`](https://github.com/gazebo-tooling/release-tools/blob/master/release.py)
+    will generate a local tarball with the source code of the new version and
+    upload it to `osrf-distributions S3`.
+ 1. `release.py` will start the following jobs in the build server
+    `build.osrfoundation.org`:
+      1. `ign/gz-fooX-debbuilder`: multiple calls for different Debian/Ubuntu releases
+      1. [`generic-release-homebrew_pull_request_updater`](https://build.osrfoundation.org/job/generic-release-homebrew_pull_request_updater/):
+      one call for Homebrew macOS release
+ 1. `build.osrfoundation.org` jobs start the work of creating releases:
+      1. `ign/gz-fooX-debbuilder`: use tarball with release sources and metadata from `ign/gz-fooX-release`
+      1. `generic-release-homebrew_pull_request_updater`: use
+         [`homebrew-simulation`](https://github.com/osrf/homebrew-simulation/)
+         repository metadata together with the release sources
+ 1. The output of the first round of initial jobs triggered by `release.py` is
+    different:
+      1. `ign/gz-fooX-debbuilder`: builds the Debian/Ubuntu .deb packages and
+         passes them to the `repository_uploader_packages` job
+      1. `generic-release-homebrew_pull_request_updater`: opens a
+         new PR to coordinate the release process in `homebrew-simulation`
+ 1. [`repository_uploader_packages`](https://build.osrfoundation.org/job/repository_uploader_packages/)
+    imports the packages created by the `ign/gz-fooX-debbuilder` job (there will be
+    one build for each platform combination of Ubuntu/Debian release
+    and architecture) and uploads the .deb packages to
+    `packages.osrfoundation.org` and [`osrf-distributions S3`](http://gazebosim.org/distributions).
+ 1. For macOS, the PR in `homebrew-simulation` waits for a comment from an
+    Gz developer with the order `build bottle` that will trigger the job
+    [`generic-release-homebrew_triggered_bottle_builder`](https://build.osrfoundation.org/job/generic-release-homebrew_bottle_builder/).
+ 1. `generic-release-homebrew_triggered_bottle_builder`will use the tarball with
+    release sources from `osrf-distributions S3` to generate the binary bottles.
+    They will be uploaded to `osrf-distributions S3`.
+
+### Prerequisites
+
+To perform a new release, a small number of configurations and credentials need to be made on the developer's system before triggering the release.
+If a permanent operating system is used for releasing, these installation steps only need to be executed once.
+
+#### Team and development checks
+
+When creating a new release, there are some guidelines to follow before starting
+the process:
+
+ * Ask the team if there are any concerns about making the release.
+ * Check if there are changes to previous library versions that need to be forward-ported.
```

Comment:
```suggestion
 * Check if there are changes to previous library versions that need to be forward-ported. Forward-ports from the intended release library version are not important.
```

Reply:
Sorry I do not understand here what we mean by "release library".